src/components/teamBoxes/TeamOneBox.tsx

```diff
@@ -67,34 +70,36 @@ const TeamOneBox = () => {
 
   // On click event for a player to be able to join the blue team-1 as a spymaster
   const joinTeamOneSpy = async () => {
-    //lines 71 - 82 are checking if the current player is already on a team
+    // lines 71 - 82 are checking if the current player is already on a team
     const teamTwo = await get(teamTwoRef);
     const teamTwoOpsAndSpys = teamTwo.val();
     let teamTwoSpymaster;
     let teamTwoOperatives;
-    //Grabbing team twos info
+    // Grabbing team twos info
     if (teamTwoOpsAndSpys && teamTwoOpsAndSpys.spymaster) {
       teamTwoSpymaster = teamTwoOpsAndSpys.spymaster;
     }
-    //Grabbing team twos info
+    // Grabbing team twos info
     if (teamTwoOpsAndSpys && teamTwoOpsAndSpys.operatives) {
       teamTwoOperatives = Object.keys(teamTwoOpsAndSpys.operatives);
     }
-    //If a player is on team 2, they cannot join this team
+    // If a player is on team 2, they cannot join this team
     if (
       (teamTwoSpymaster && teamTwoSpymaster.playerId === playerId) ||
       (teamTwoOperatives && teamTwoOperatives.includes(playerId))
     ) {
+      // tslint:disable-next-line:no-console
```

Comment:
not sure - is this doing something here?

Reply:
For some reason typescript gives me a warning on console logs so this is just me disabling the rule but I can remove them!